code/modules/mob/abstract/new_player/character_traits.dm

```diff
@@ -79,3 +79,29 @@
 	name = "Major Hemophilia"
 	desc = "Your blood lacks ALL clotting factors, causing wounds to never stop bleeding."
 	trait_type = TRAIT_DISABILITY_HEMOPHILIA_MAJOR
+
+
+ABSTRACT_TYPE(/datum/character_disabilities/organ_scarring)
+	name = "Organ Scarring"
+	var/affected_organ
+
+/datum/character_disabilities/organ_scarring/apply_self(var/mob/living/carbon/human/target)
+	var/obj/item/organ/internal/affecting = target.internal_organs_by_name[affected_organ]
+	if(affecting)
+		affecting.max_damage = affecting.max_damage*0.5
```

Comment:
i'm not 100% sure, but i think this should set it using set_max_damage()

![image](https://github.com/user-attachments/assets/08ccbfe5-0038-49ad-b8ff-fecbde9cff0e)